src/tools/perf/lib/libperf.c

```diff
@@ -1486,7 +1486,8 @@ static ucs_status_t uct_perf_setup(ucx_perf_context_t *perf)
     uct_iface_params_t iface_params = {
         .field_mask           = UCT_IFACE_PARAM_FIELD_OPEN_MODE   |
                                 UCT_IFACE_PARAM_FIELD_STATS_ROOT  |
-                                UCT_IFACE_PARAM_FIELD_RX_HEADROOM,
+                                UCT_IFACE_PARAM_FIELD_RX_HEADROOM |
+                                UCT_IFACE_PARAM_FIELD_DEVICE,
```

Comment:
good fix, was the issue found with this PR?

Reply:
Yes, test `cuda_copy/test_uct_loopback_cuda.envelope/0` failed  
https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=67174&view=logs&j=f6a00158-c162-5d05-e321-5b7f75412dc2&t=3041e05d-2adc-5ccc-5b20-437cf039c2a1&l=21430
, after I addressed the comment https://github.com/openucx/ucx/pull/9264#discussion_r1283333535.